LoggerContext not initiating in docker env

I am consistently getting the following and then no more logs from `bitcoin-s-walletserver`
```
bitcoin-s-walletserver-1  | Removing jre as its not linked correctly on ARM machines, see https://github.com/bitcoin-s/bitcoin-s/issues/4369!
bitcoin-s-walletserver-1  | [DEBUG] [09/06/2022 14:49:28.036] [main] [EventStream] StandardOutLogger started
bitcoin-s-walletserver-1  | Failed to instantiate [ch.qos.logback.classic.LoggerContext]
bitcoin-s-walletserver-1  | Reported exception:
bitcoin-s-walletserver-1  | ch.qos.logback.core.LogbackException: Failed to initialize Configurator: ch.qos.logback.classic.util.DefaultJoranConfigurator using ServiceLoader
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:96)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:77)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.spi.LogbackServiceProvider.initializeLoggerContext(LogbackServiceProvider.java:50)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.spi.LogbackServiceProvider.initialize(LogbackServiceProvider.java:41)
bitcoin-s-walletserver-1  | 	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:152)
bitcoin-s-walletserver-1  | 	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:139)
bitcoin-s-walletserver-1  | 	at org.slf4j.LoggerFactory.getProvider(LoggerFactory.java:421)
bitcoin-s-walletserver-1  | 	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:407)
bitcoin-s-walletserver-1  | 	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:356)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.Logger$.apply(Slf4jLogger.scala:32)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.SLF4JLogging.log(Slf4jLogger.scala:20)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.SLF4JLogging.log$(Slf4jLogger.scala:20)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.Slf4jLogger.log$lzycompute(Slf4jLogger.scala:56)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.Slf4jLogger.log(Slf4jLogger.scala:56)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.Slf4jLogger$$anonfun$receive$1.applyOrElse(Slf4jLogger.scala:107)
bitcoin-s-walletserver-1  | 	at akka.actor.Actor.aroundReceive(Actor.scala:537)
bitcoin-s-walletserver-1  | 	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
bitcoin-s-walletserver-1  | 	at akka.event.slf4j.Slf4jLogger.aroundReceive(Slf4jLogger.scala:56)
bitcoin-s-walletserver-1  | 	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:580)
bitcoin-s-walletserver-1  | 	at akka.actor.ActorCell.invoke(ActorCell.scala:548)
bitcoin-s-walletserver-1  | 	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
bitcoin-s-walletserver-1  | 	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
bitcoin-s-walletserver-1  | 	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
bitcoin-s-walletserver-1  | 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
bitcoin-s-walletserver-1  | 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
bitcoin-s-walletserver-1  | 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
bitcoin-s-walletserver-1  | 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
bitcoin-s-walletserver-1  | 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
bitcoin-s-walletserver-1  | Caused by: java.time.zone.ZoneRulesException: Unknown time-zone ID: SSXXX
bitcoin-s-walletserver-1  | 	at java.base/java.time.zone.ZoneRulesProvider.getProvider(ZoneRulesProvider.java:280)
bitcoin-s-walletserver-1  | 	at java.base/java.time.zone.ZoneRulesProvider.getRules(ZoneRulesProvider.java:235)
bitcoin-s-walletserver-1  | 	at java.base/java.time.ZoneRegion.ofId(ZoneRegion.java:121)
bitcoin-s-walletserver-1  | 	at java.base/java.time.ZoneId.of(ZoneId.java:410)
bitcoin-s-walletserver-1  | 	at java.base/java.time.ZoneId.of(ZoneId.java:358)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.pattern.DateConverter.start(DateConverter.java:45)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.pattern.ConverterUtil.startConverters(ConverterUtil.java:37)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.pattern.PatternLayoutBase.start(PatternLayoutBase.java:90)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.encoder.PatternLayoutEncoder.start(PatternLayoutEncoder.java:28)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.ImplicitModelHandler.postHandleComplex(ImplicitModelHandler.java:209)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.ImplicitModelHandler.postHandle(ImplicitModelHandler.java:187)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.DefaultProcessor.secondPhaseTraverse(DefaultProcessor.java:248)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.DefaultProcessor.secondPhaseTraverse(DefaultProcessor.java:244)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.DefaultProcessor.secondPhaseTraverse(DefaultProcessor.java:244)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.DefaultProcessor.traversalLoop(DefaultProcessor.java:90)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.model.processor.DefaultProcessor.process(DefaultProcessor.java:106)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.joran.GenericXMLConfigurator.processModel(GenericXMLConfigurator.java:199)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.joran.GenericXMLConfigurator.doConfigure(GenericXMLConfigurator.java:165)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.joran.GenericXMLConfigurator.doConfigure(GenericXMLConfigurator.java:122)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.core.joran.GenericXMLConfigurator.doConfigure(GenericXMLConfigurator.java:65)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.util.DefaultJoranConfigurator.configureByResource(DefaultJoranConfigurator.java:53)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.util.DefaultJoranConfigurator.configure(DefaultJoranConfigurator.java:34)
bitcoin-s-walletserver-1  | 	at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:90)
bitcoin-s-walletserver-1  | 	... 27 more
bitcoin-s-walletserver-1  | [DEBUG] [09/06/2022 14:49:28.282] [main] [EventStream(akka://bitcoin-s-server-1662475767731)] logger log1-Slf4jLogger started
bitcoin-s-walletserver-1  | [DEBUG] [09/06/2022 14:49:28.284] [main] [EventStream(akka://bitcoin-s-server-1662475767731)] Default Loggers started
```

--- a/app/oracle-server/src/main/resources/logback.xml
+++ b/app/oracle-server/src/main/resources/logback.xml
@@ -1,7 +1,7 @@
 <configuration scan="true" scanPeriod="15 seconds" >
     <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%date{yyyy-MM-dd'T'HH:mm:ss,SSXXX, UTC}UTC %level [%logger{0}] %msg%n</pattern>
+            <pattern>%date{ISO8601,UTC}UTC %level [%logger{0}] %msg%n</pattern>
         </encoder>
     </appender>
 

--- a/app/scripts/src/main/resources/logback.xml
+++ b/app/scripts/src/main/resources/logback.xml
@@ -1,7 +1,7 @@
 <configuration scan="true" scanPeriod="15 seconds" >
     <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%date{yyyy-MM-dd'T'HH:mm:ss,SSXXX, UTC}UTC %level [%logger{0}] %msg%n</pattern>
+            <pattern>%date{ISO8601,UTC}UTC %level [%logger{0}] %msg%n</pattern>
         </encoder>
     </appender>
 
@@ -23,7 +23,7 @@
             <totalSizeCap>2GB</totalSizeCap>
         </rollingPolicy>
         <encoder>
-            <pattern>%date{yyyy-MM-dd'T'HH:mm:ss,SSXXX, UTC}UTC %level [%logger{0}] %msg%n</pattern>
+            <pattern>%date{ISO8601,UTC}UTC %level [%logger{0}] %msg%n</pattern>
         </encoder>
     </appender>
 

--- a/app/server/src/main/resources/logback.xml
+++ b/app/server/src/main/resources/logback.xml
@@ -1,7 +1,7 @@
 <configuration scan="true" scanPeriod="15 seconds" >
     <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%date{yyyy-MM-dd'T'HH:mm:ss,SSXXX, UTC}UTC %level [%logger{0}] %msg%n</pattern>
+            <pattern>%date{ISO8601,UTC}UTC %level [%logger{0}] %msg%n</pattern>
         </encoder>
     </appender>
 

